src/sdt.rs

```diff
@@ -47,7 +48,8 @@ impl SdtHeader {
 
         // Check that the lowest byte is 0
         if sum % 0b1111_1111 != 0 {
-            return Err(AcpiError::SdtInvalidChecksum);
+            panic!("{}\n", sum)
```

Comment:
I'm not sure the library should ever panic; it seems like it takes the choice away from the user of whether they want to carry on with invalid ACPI tables

Reply:
Ooops, I was just debugging with panics haha, should not be here :roll_eyes: